src/renderer/Generics/redesign/Toggle/Toggle.tsx

```diff
@@ -46,12 +51,11 @@ export const Toggle = ({
           type: 'checkbox',
           role: 'switch',
           id: 'flexSwitchCheckDefault',
-          defaultChecked: isChecked,
+          checked: isChecked,
```

Comment:
I think defualtChecked may need to be set so that the component starts as a "controlled component". Not 100% sure. Check the warning in the console when opening the "Notifications preferences.." for the first time in app

```instrument.js:105 Warning: A component is changing an uncontrolled input to be controlled. This is likely caused by the value changing from undefined to a defined value, which should not happen. Decide between using a controlled or uncontrolled input element for the lifetime of the component. More info: https://reactjs.org/link/controlled-components```